### ISSUE TITLE
Compile does not find extracted directory after unpacking

### ISSUE THREAD

**Issue by dxmann** [gobolinux/Compile#34]
Hello,
I've encountered this issue updating the bullet recipe.
The saved log says:
```
Compile: Unpacking file /Data/Compile/Archives/bullet-2.87.tar.gz...
Compile: Directory /Data/Compile/Sources/bullet-2.87 not found.
```
The recipe is
```
# Recipe (MakeRecipe) for Bullet by Filipe Vieira, on Fri Jan 18 20:42:37 WET 2008
compile_version=1.10.0
url="https://github.com/bulletphysics/bullet3/archive/2.87.tar.gz"
file_size=56691047
file_md5=7566fc00d925a742e6f7ec7ba2d352de
file="bullet-2.87.tar.gz"
recipe_type=configure
autogen_before_configure=yes

pre_link()
{
   # Enlightenment 18 will ask for the missing headers below
   for i in ./src/BulletSoftBody/btSoftBodySolvers.h \
            ./src/BulletSoftBody/btDefaultSoftBodySolver.h \
            ./src/BulletSoftBody/btSoftBodySolverVertexBuffer.h \
   do
      cp -v $i $target/include/bullet/BulletSoftBody
   done
              
   cp -v ./src/BulletDynamics/ConstraintSolver/btFixedConstraint.h \
      $target/include/bullet/BulletDynamics/ConstraintSolver
}
```

**Comment by ermo** [comment]
The reason it doesn't work is that the argument to the `file` variable needs to begin with `bullet3`. =)

The fixed, updated-to-2.89 Recipe builds and looks like this:

```
# Recipe (MakeRecipe) for Bullet by Filipe Vieira, on Fri Jan 18 20:42:37 WET 2008
compile_version=017-GIT
url="https://github.com/bulletphysics/bullet3/archive/2.89.tar.gz"
file_size=135742506
file_md5=d239b4800ec30513879834be6fcdc376
file="bullet3-2.89.tar.gz"
recipe_type=cmake
#autogen_before_configure=yes

cmake_options=(
   OpenGL_GL_PREFERENCE="GLVND"   
)

pre_link()
{
   # Enlightenment 18 will ask for the missing headers below
   for i in ./src/BulletSoftBody/btSoftBodySolvers.h \
            ./src/BulletSoftBody/btDefaultSoftBodySolver.h \
            ./src/BulletSoftBody/btSoftBodySolverVertexBuffer.h \
   do
      cp -v $i $target/include/bullet/BulletSoftBody
   done
              
   cp -v ./src/BulletDynamics/ConstraintSolver/btFixedConstraint.h \
      $target/include/bullet/BulletDynamics/ConstraintSolver
}
```

**Comment by lucasvr** [comment]
Thanks @ermo! I've pushed the updated recipe now. @dxmann, it should work as expected now. Please reopen if you still face any problems with this recipe.
